nexus/proposals/tests/test_models.py

```diff
@@ -0,0 +1,45 @@
+# Standard Stuff
```

Comment:
Is this required? Since there's no Standard Stuff, I think we can skip this.

Reply:
Fixed it. Thanks to point out :)